xml/deploy_cephadm.xml

```diff
@@ -646,6 +646,14 @@ o- time_server ................................................ [enabled]
       <command>man 1 skopeo</command>.
      </para>
     </tip>
+    <procedure>
+      <step>
+        <para>
+          Install <command>podman</command> and <command>skopeo</command>:
+        </para>
+<screen>&prompt.smaster;zypper install podman &amp;&amp; skopeo</screen>
```

Comment:
I thought that the way to install multiple packages with Zypper was just to list them separated by spaces?

`zypper in package1 package2`

If you type:

`zypper in package_1 && package_2`

then as I understand it that will **install** `package_1` and if that is successful it will **run** `package_2`.

Reply:
Easy as :)